nano/node/wallet.cpp

```diff
@@ -1692,7 +1692,8 @@ void nano::wallets::ongoing_compute_reps ()
 {
 	compute_reps ();
 	auto & node_l (node);
-	auto compute_delay (network_params.network.is_dev_network () ? std::chrono::milliseconds (10) : std::chrono::milliseconds (15 * 60 * 1000)); // Representation drifts quickly on the test network but very slowly on the live network
```

Comment:
The test network is supposed to be identical to the live network except for the genesis block.
In reality, it cannot be identical but we want it to be as close to the live as possible.
This change seems unnecessary for the general test network, it is only useful when we use the test network overrides to create local private networks. I think we should rethink this.

I think the proper solution is to create a new type of network that is specifically for local private networks.
I described something like that on discord a few month ago (the open network), before the attacks.

In the short term, we could have an ENVIRONMENT variable to override this behaviour.

Reply:
Right, although the check itself is cheap and fast, there is no need for non-local nodes to do this check every second. Creating a new type of network would be ideal, but for that we would need to create some kind of a network settings file, the current nested conditionals approach is neither pretty nor scalable. I remember you writing about it on discord, was there any concrete design agreed upon? I could look into implementing that, it seems it's mostly refactoring and extracting magical constants to configuration files.